Realm/ObjectServerTests/RealmServer.swift

```diff
@@ -922,6 +930,97 @@ public class RealmServer: NSObject {
         return appId
     }
 
+    public func retrieveSyncServiceId(appServerId: String) throws -> String {
+        guard let session = session else {
+            throw URLError(.unknown)
```

Comment:
These should be `fatalError()`, as the session will only be `nil` due to writing invalid code calling this (specifically, trying to call it in a child process).